extensions/sd_api_pictures/script.py

```diff
@@ -64,38 +105,46 @@ def input_modifier(string):
 # Get and save the Stable Diffusion-generated picture
 def get_SD_pictures(description):
 
-    global params, pic_id
+    global params
+
+    if params['manage_VRAM']: give_VRAM_priority('SD')
 
     payload = {
         "prompt": params['prompt_prefix'] + description,
         "seed": -1,
         "sampler_name": "DPM++ 2M Karras",
         "steps": 32,
         "cfg_scale": 7,
-        "width": params['side_length'],
-        "height": params['side_length'],
+        "width": params['width'],
+        "height": params['height'],
         "restore_faces": params['restore_faces'],
         "negative_prompt": params['negative_prompt']
     }
     
-    response = requests.post(url=f'{params["address"]}/sdapi/v1/txt2img', json=payload)
+    print(f'Prompting the image generator via the API on {params["address"]}…')
```

Comment:
Minor, but `…` is a unicode symbol that I'm not sure will render in all terminals (vs just normal `...`)